test/codegen/end_to_end_test.dart

```diff
@@ -121,6 +121,10 @@ void main(List<String> args) {
     runQuery('K', new QueryOptions(
         includeInherited: true, withAnnotations: [lib.getType('AnnotC')]));
 
+    runQuery('J', new QueryOptions(includeMethods: true));
+    runQuery('J2', new QueryOptions(
```

Comment:
We could a second test for it (it would require 2 more classes): you'd keep the current test as is, but for the second test we'd be able to see that the generated static configuration doesn't include the member of the base class.